<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowDisplay</class>
 <widget class="QMainWindow" name="WindowDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WindowDisplay</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="machineLayout">
        <item>
         <layout class="QVBoxLayout" name="machineControlsLayout">
          <item>
           <widget class="QGraphicsView" name="displayView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="controlButtonsLayout">
            <item>
             <widget class="QPushButton" name="onButton">
              <property name="text">
               <string>ON</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="resetButton">
              <property name="text">
               <string>RESET</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rebootButton">
              <property name="text">
               <string>REBOOT</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="slotsLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="slotsLabel">
            <property name="text">
             <string>Slots</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="auxLayout">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>AUX</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_19">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_18">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="slot1Layout">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_17">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_16">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="slot2Layout">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_15">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_14">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="slot3Layout">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_13">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_12">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="slot4Layout">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="slot6Layout_2">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_11">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_10">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="slot6Layout">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>6</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="slot7Layout">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>7</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="memoryLayout">
        <item>
         <widget class="QLabel" name="codeLabel">
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="codeView"/>
        </item>
        <item>
         <widget class="QLabel" name="memoryLabel">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="memoryView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="stackCodeLayout">
        <item>
         <widget class="QLabel" name="stackLabel">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="stackView"/>
        </item>
        <item>
         <widget class="QLabel" name="registersLabel">
          <property name="text">
           <string>Registers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="registersView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="debuggerLayout">
          <item>
           <widget class="QPushButton" name="runButton">
            <property name="text">
             <string>RUN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stepButton">
            <property name="text">
             <string>STEP</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stack</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Memory</string>
   </property>
  </action>
  <action name="actionDisassembly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disassembly</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
